packages/react-components/react-toast/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { renderAlert_unstable } from './renderAlert';
+import { useAlert_unstable } from './useAlert';
+import { useAlertStyles_unstable } from './useAlertStyles.styles';
+
+import type { AlertProps } from './Alert.types';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+
+/**
+ * An Alert component displays a brief, important message to attract a user's attention
+ *  without interrupting their current task.
+ */
+export const Alert: ForwardRefComponent<AlertProps> = React.forwardRef((props, ref) => {
```

Comment:
will port the stories and documentation in a follow up PR, this package is still private so there's no rush